src/controllers/room.controller.js

```diff
@@ -0,0 +1,30 @@
+const roomService = require('../services/room.service');
+
+const create = async (req, res) => {
+  const { name } = req.body;
+  const newRoom = await roomService.create(name);
+
+  res.status(201).send(newRoom);
+};
+
+const remove = async (req, res) => {
+  const { id } = req.params;
+
+  await roomService.remove(id);
```

Comment:
Consider adding error handling for the asynchronous operation. If the roomService.remove method fails, it could throw an error that should be caught and handled appropriately.